src/app.ts

```diff
@@ -10,9 +12,10 @@ app.use(bodyParser.json());
 app.use(validateUserId);
 
 app.use('/api', userRoutes);
+app.use('/api', artistRoutes);
+app.use('/api', albumRoutes);
 
-app.use((err: any, req: Request, res: Response, next: NextFunction) => {
-  console.error(err.stack);
+app.use((err: any, req: Request, res: Response) => {
```

Comment:
Please avoid using `any` all cases. For this particular one, you may create custom type extend `Error` or use an existing error types (if any)